uportal-war/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp

```diff
@@ -139,8 +139,8 @@
         </a>
         <c:forEach var="image" items="${thumbnailImages}" varStatus="status">
             <a href="#">
-                <img src="${image}" />
-                <span class="caption">Background ${status.index + 1}</span>
+                <img src="${image}" alt=${imageCaptions[status.index]}/>
+                <span class="caption">${imageCaptions[status.index]}</span>
```

Comment:
and because there is just after `<span class="caption">the description of the image</span>` everything is ok.
even a blind user can change his own background image...